terraform-plans/modules/GitHub/workflows/main.tf

```diff
@@ -9,36 +9,76 @@ terraform {
 
 provider "github" {
   owner = var.owner
-  app_auth {} # using environmet variables for auth
+  app_auth {} # using environment variables for auth
 }
 
```

Comment:
These should not be required, since module will inherit this provider from root module, can you try without it?

Reply:
I guess it's because we want to use `owner = var.owner`?

Reply:
it will also be inherited

Reply:
I am not very sure about this part. I tend to keep it.